The-SMIRNOFF-force-field-format.md

```diff
@@ -218,6 +216,13 @@ For example, to ensure water molecules are assigned partial charges for [TIP3P](
 </LibraryCharges>
 ```
 
+
```

Comment:
Are the `<LibraryCharges>` examples above still accurate?

Reply:
Wow. They actually didn't parse. Alanine was missing a `/` at the end of the LibraryCharge line, and TIP3P failed because `+0.417` doesn't parse with the `+`, so I've change it to just be `0.417`.

Fixed and added a test, with the `TODO`:
```
        # TODO: This test is practically useless while the XML strings are hard-coded at the top of this file.
        #       We should implement something like doctests for the XML snippets on the SMIRNOFF spec page.
```